mkdocs.yml

```diff
@@ -0,0 +1,69 @@
+---
+nav:
+    - Home: index.md
+    - Installation: installation/index.md
+    - Getting Started:
+          - 1. Initial Steps: getting-started/initial-steps.md
+          - 2. Create a Component: getting-started/create-component.md
+          - 3. Use the Template Tag: getting-started/reference-component.md
+          - 4. Render Your View: getting-started/render-view.md
+          - 5. Learn More: getting-started/learn-more.md
```

Comment:
These pages seem pretty short. Could these be combined into one "Getting Started" page?

Reply:
I tried that initially. But I had several devs read through these docs to give opinions, and they strongly felt it was too busy on one page.

I personally think it should remain separated for readability. Bite-sized chunks are easier to digest compared to ~3 screen heights of scrolling.

Reply:
Sounds good. Especially if you got feedback suggesting that.